symengine/tests/test_lambdify.py

```diff
@@ -46,38 +46,40 @@ def allclose(vec1, vec2, rtol=1e-13, atol=1e-13):
 def test_Lambdify():
     n = 7
     args = x, y, z = se.symbols('x y z')
-    l = se.Lambdify(args, [x+y+z, x**2, (x-y)/z, x*y*z], backend='lambda')
-    assert allclose(l(range(n, n+len(args))),
+    L = se.Lambdify(args, [x+y+z, x**2, (x-y)/z, x*y*z], backend='lambda')
```

Comment:
It is mentioned [here](https://www.python.org/dev/peps/pep-0008/#names-to-avoid).